stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1234,6 +1249,11 @@ public final class TestSuite {
     case .parameterized(code: let code, _):
       code(parameter!)
     }
+
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
```

Comment:
This build configuration has to be passed on the swiftc command line when building the library.  I don't think we are doing this currently.